CHANGELOG.md

```diff
@@ -1,3 +1,75 @@
+# #develop#
+
+Replace this text with a summarized description of this release's contents.
+## Breaking changes
+
+Replace this explanations for how to resolve the breaking changes.
+## New deprecations
+
+Replace this text with links to deprecation guides.
+## New features
```

Comment:
For dialogs, we added an "orderVerticalActionsByEmphasis" flag which orders buttons by emphasis in vertical layout, and fixed a couple of bugs.

Reply:
Since the `orderVerticalActionByEmphasis` flag only exist on a _private_ header file I don't think we should call it out.